website/cue/reference/cli.cue

```diff
@@ -261,15 +261,16 @@ cli: {
 
 		"tap": {
 			description: """
-				Observe log events from topology components.
+				Observe output log events from source or transform components. Logs are sampled
+				at a specified interval.
 				"""
 
 			flags: _default_flags
 
 			options: {
 				"interval": {
 					_short:      "i"
-					description: "Interval to sample metrics at, in milliseconds"
+					description: "Interval to sample logs at, in milliseconds"
```

Comment:
Thanks for updating these! Could you update the CLI help text too to match (`vector tap --help`)?